### ISSUE TITLE
can i get the from language in any way ?

### ISSUE THREAD

**Issue by NatHsn26** [nidhaloff/deep-translator#43]
hello!
im coding something for myself and i have someneed i wish you to informate me :)
so in first , is it possible to autodetect the language source but to recuperate the language source name in any case ?
and secundo is it possible to :
i give a word random, its give me the translation if different from the Input . but iwould its continue search traduction for this word while the trad is something identic. but only in autodetectlanguage and give me the result when its not the same than input .Thanks you 🦋

**Comment by nidhaloff** [comment]
@NatHsn26 Hi ;) I'm not sure I understand what you mean. If you get a translation for a word, then it will be probably not the same as the source. If yes then the translation would be wrong. In this case why would you try and translate it again? 

Can you clarify more what you are trying to do

**Comment by NatHsn26** [comment]
Good evening !Yes sure let's talk later im tired of my day its late where i am living. Have a great day :)

**Comment by NatHsn26** [comment]
Hello, i dont get enough time for myself to check all my conversation I got problem at home , all my excuses really.
Well im not native english but if i understand good what i wrote it’s dont mean anything lol 📚🌪 so i reformule again ;

Im for the moment using an other translator.
First problem i got is i cant get the language detected , to say for example, the word mean « » in french and input was given in Chinese. I solved this problem with IsDetailResult , and by browsing the huge result to search the abbreviation in that string .
-I was first wondering if your framework is able to do the same ? 

And finally that is the incomprehensible part of my first message : 

i would get the translation in french and know the language source , but i would also know this language source to again find a translation for the first input, by excepting the first detected_lang.

So its could say me that a word have been detected as english and mean « », run again but dont count english so its’d say me it also a Hungarian word and mean « »... and all this while the result reach an ‘unknow error’(No more language source detected i mean so).

on the other translators i am using with python ,when isdetailresult is True , even if the word mean nothing , its will fullfill with a «random? » language to try it , but it seem there is no way to test other language with autodetect,by using an except language .

i hope it’s understandable, 🤝
